tests/e2e/e2e_setup_test.go

```diff
@@ -267,18 +316,26 @@ func (s *IntegrationTestSuite) runIBCRelayer() {
 	s.connectIBCChains()
 }
 
-func (s *IntegrationTestSuite) configureChain(chainId string) {
+func (s *IntegrationTestSuite) configureChain(chainId string, validatorConfigs []*chain.ValidatorConfig) {
+
 	s.T().Logf("starting e2e infrastructure for chain-id: %s", chainId)
 	tmpDir, err := ioutil.TempDir("", "osmosis-e2e-testnet-")
 
 	s.T().Logf("temp directory for chain-id %v: %v", chainId, tmpDir)
 	s.Require().NoError(err)
 
+	b, _ := json.Marshal(validatorConfigs)
+	file := fmt.Sprintf("%v/%v-configEncode", tmpDir, chainId)
```

Comment:
As per my previous comment, instead of creating a file and writing to disk, we could pass in the byte stream via CLI

Reply:
As in, pass it in as a flag via Cmd via dockertest?

Reply:
Yes